setup.py

```diff
@@ -44,6 +44,8 @@
     librdkafka_libname = 'rdkafka'
 
 module = Extension('confluent_kafka.cimpl',
+                   include_dirs=['/usr/local/include/'],
+                   library_dirs=['/usr/local/lib'],
```

Comment:
this isn't portable (especially for M1 where brew's librdkafka will be in `/opt` and not `/usr/local`)

you can usually utilize `CPATH="$(brew --prefix librdkafka)/include" LDFLAGS="-L$(brew --prefix librdkafka)/lib" ...` to build a wheel